js-intro.js

```diff
@@ -1,33 +1,58 @@
-// ASSESSMENT 1: INTRO TO JAVASCRIPT
-// Coding practical questions
+// // ASSESSMENT 1: INTRO TO JAVASCRIPT
+// // Coding practical questions
 
-// Please read all questions thoroughly
-// Psuedo coding is highly recommended
+// // Please read all questions thoroughly
+// // Psuedo coding is highly recommended
 
-// --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Farhenheit.
-// Use the test variables provided below. Expected outcome: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
+// // --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Farhenheit.
+// // Use the test variables provided below. Expected outcome: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
+// //create a function called checkBoilingPoint
+ checkBoilingPoint = (temp) => {
+// function checkBoilingPoint (temperature){
+    if(temp === 212){
+        return temp + " is at boiling point"
+    }else if(temp < 212){
+        return temp + " is below boiling point"
+    }else {
+        return temp + " is above boiling point"
+    }
+}
+console.log(checkBoilingPoint(temp1));
+console.log(checkBoilingPoint(temp2));
+console.log(checkBoilingPoint(temp3));
 
 
 
-// --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
-// Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
+// // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
+// // // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
 
 var myNumbers1 = [3, 7, 0, 6, -9]
 
+const mult5 = (array) => {
+    let newArr = []
+    for(let i=0; i<array.length; i++){
+        newArr.push(array[i] * 5)
+    }
+    return newArr
+}
 
+console.log(mult5(myNumbers1));
 
-
-// --------------------3) Create a function that multiplies each number in the array by 5 using map.
-// Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
+// // --------------------3) Create a function that multiplies each number in the array by 5 using map.
+// // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
 var myNumbers2 = [8, -7, 0, 6, 2]
 
-
+multi5 = (array) => {
+    let newArr = array.map(value => value * 5)
+    return newArr
+}
```

Comment:
You can save yourself a line of code by returning your mapped array outright, rather than creating a new variable in order to turn around and immediately return it. Map is an accessor which doesn't permanently change to original array, so if you wanted to use those values later in your code you would need to save it to a new variable like you did. But for this purpose, you can save yourself some key strokes :)